tests/test_plugins.py

```diff
@@ -134,3 +134,9 @@ def test_timestamped_geo_json(self):
         mape = folium.Map([47, 3], zoom_start=1)
         mape.add_children(plugins.TimestampedGeoJson(data))
         mape._repr_html_()
+
+    def test_heat_map(self):
+        data = (np.random.normal(size=(100,2))*np.array([[1,1]])+np.array([[48,5]])).tolist()
+        mapa = folium.Map([48., 5.], tiles='stamentoner', zoom_start=6)
+        mapa.add_children(plugins.HeatMap(data))
+        mapa._repr_html_()
```

Comment:
This test only if the HTML is created, correct? I guess that all the plugins are tested like that...  Not sure if we can do better.